variables.go

```diff
@@ -176,13 +178,14 @@ func (e *Executor) handleDynamicVariableContent(v Var) (string, error) {
 // variables in almost all properties using the Go template package
 func (t *Task) ReplaceVariables(vars Vars) (*Task, error) {
 	r := varReplacer{vars: vars}
+
 	new := Task{
 		Desc:      r.replace(t.Desc),
 		Sources:   r.replaceSlice(t.Sources),
 		Generates: r.replaceSlice(t.Generates),
 		Status:    r.replaceSlice(t.Status),
 		Dir:       r.replace(t.Dir),
-		Vars:      r.replaceVars(t.Vars),
+		Vars:      nil, // no longer needed.
```

Comment:
I remove this, because I am afraid it's just a source of confusion when it's no longer used. If we where to set it to _anything_, it should probably be the full, passed in `vars`.